routes/rendering-info/html-static.js

```diff
@@ -58,27 +64,32 @@ module.exports = {
           name: styleHashMap.default
         }
       ],
+      sophieModules: [],
       markup: staticTemplate.render(renderingData)
-    }
+    };
 
     // add sophie viz color module to stylesheets in response iff necessary
     let isSophieVizColorDefined = false;
     let parties = request.payload.item.parties;
     if (parties !== undefined) {
       parties.forEach(party => {
         let vizPattern = /^s-viz-color-party.*/;
-        if (_.has(party, 'color.classAttribute') && vizPattern.test(party.color.classAttribute)) {
+        if (
+          _.has(party, "color.classAttribute") &&
+          vizPattern.test(party.color.classAttribute)
+        ) {
           isSophieVizColorDefined = true;
         }
-      })
+      });
     }
 
     if (isSophieVizColorDefined) {
-      data.stylesheets.push({
-        url: 'https://service.sophie.nzz.ch/bundle/sophie-viz-color@^1[parties].css'
+      data.sophieModules.push({
+        name: "sophie-viz-color@^1",
```

Comment:
Same here. You can just use `sophie-viz-color@1`. So it will always use the newest 1 release